packages/flutter/lib/src/sentry_flutter_options.dart

```diff
@@ -291,6 +291,22 @@ class SentryFlutterOptions extends SentryOptions {
   /// Screen content masking is enabled by default.
   final privacy = SentryPrivacyOptions();
 
+  /// Sets the path to the Sentry Database Directory used in the Sentry Native SDK.
+  ///
+  /// **Default behavior**: When `null` (the default), the path defaults to
+  /// `.sentry-native` in the current working directory (CWD).
+  ///
+  /// **Important**: While using the default is convenient for development, we
+  /// strongly recommend providing an explicit database path for production
+  /// deployments to ensure predictable behavior.
+  ///
+  /// This option is only used on platforms that utilize the Sentry Native SDK directly,
+  /// specifically Desktop Linux and Windows.
```

Comment:
Should we also mention here which paths are set by default on those platforms?

Reply:
the default is `.sentry-native` or do you think its unclear in the comment?